edk2toolext/edk2_invocable.py

```diff
@@ -61,6 +61,10 @@ def RetrieveCommandLineOptions(self, args: object) -> None:
         '''  Implement in subclass to retrieve command line options from the argparser namespace '''
         pass
 
+    def GetSkippedDirectories(self) -> Tuple[str]:
```

Comment:
this is implemented in base_abstract_invocable class.  Why needed here for same implementation?

Reply:
I just forgot to remove after adding it in the abstract class. I'll update it to be removed.